### ISSUE TITLE
SpaceX compatibility

### ISSUE THREAD

**Issue by ZXQWAS** [kingarthur91/PyCoalTBaA#4]
At https://forums.factorio.com/viewtopic.php?f=190&t=51967&p=435831#p435831 you didn't mention SpaceX. Don't know what was in there specifically yet, but please don't drop it. 
Also there's this mod for SpaceX compatibility https://mods.factorio.com/mod/ERPTbaAB , but SuperSandro isn't much motivated to maintain it. There's currently a problem with py rare earth in it, which was supposed to be handled. Maybe you'll have some crossovers.

**Comment by kingarthur91** [comment]
not going to drop spaceX. that was just a first run priority list. 

i dont know if ill do anything with ERTbaAB if hes not planning on maintaining it and given that im re-balancing the hell out of py+angel and abpy it probably wont fit well anyway.

ill look into though and come up with something.

**Comment by ZXQWAS** [comment]
I've looked into the recipes somewhat and see that it could work just fine. Rare earth is already added to ground fabricator station, if molybdenum and kimberlite were added to it or dropships as well, it'd be nice. Just a couple lines of code. This is not final as I'm nowhere near space science yet.
A yellow belt of circuit 2 seems to make SpaceX with addons possible if left overnight for a week or so. Balancing some research costs if Py mods are present could be a good idea too though.
